src/listeners/interactionCreate.ts

```diff
@@ -136,8 +137,16 @@ client.on("interactionCreate", async interaction => {
 								console.error(`Couldn't get string ${path} in English for ${file}, please fix this`)
 						}
 					}
-					if (typeof string === "string" && variables)
+					if (typeof string === "string" && variables) {
 						for (const [variable, text] of Object.entries(variables)) string = string.replace(`%%${variable}%%`, String(text))
+						const discordLocale = transformBotLocale(lang) ?? "en-GB"
```

Comment:
why wouldn't this support all languages?

Reply:
If the locale is `en` then `transformBotLocale` will return `null`

Reply:
That doesn't answer my question. Why doesn't this support all the languages on the bot?